### PR TITLE
Full website validation errors

### PR DESCRIPTION
**Pull request by jgil-r**
Fixed validation errors in install/error/cli/benchmark pages. #335

### CODE REVIEW

File: website/public/benchmarks/criterion-json-to-html.mjs [hunk 1]
```diff
@@ -69,9 +69,15 @@ export function makeBenchmarkHTML(seriess) {
       )}
 
       <div class="x-axis-label">response time (lower is better)</div>
+      <!-- 
+        title attribute must be broken into new line
+        because the html spec treats &#013; as a parse error, which
+        gives us validation errors.
+      -->
```
Comment (strager) [inline website/public/benchmarks/criterion-json-to-html.mjs:76]: Nit: No need for this comment. Delete it.

File: website/public/benchmarks/criterion-json-to-html.mjs [hunk 2]
```diff
@@ -69,9 +69,15 @@ export function makeBenchmarkHTML(seriess) {
...
       <label
         class="log-scale javascript-only"
-        title="checked: logarithmic X axis (emphasizing ratios).&#013;unchecked: linear X axis (emphasizing time)."
+        title="checked: logarithmic X axis (emphasizing ratios).
+        unchecked: linear X axis (emphasizing time)."
```
Comment (strager) [inline website/public/benchmarks/criterion-json-to-html.mjs:80]: Blocking: This adds a bunch of spaces before the word 'unchecked':
Screen Shot 2021-06-04 at 00 14 53

It's supposed to be flush with the left side:
Screen Shot 2021-06-04 at 00 15 09

File: website/public/install/index.ejs.html [hunk 3]
```diff
@@ -1540,7 +1544,6 @@ <h4 id="manual-vim-pathogen">Install with Pathogen</h4>
           This option requires
           <a href="https://github.com/tpope/vim-pathogen">Pathogen</a>.
         </p>
-        <p />
```
Comment (strager) [inline website/public/install/index.ejs.html:1543]: Heh, how'd that get there? xD

**Review by strager — CHANGES_REQUESTED** [review]

File: website/public/benchmarks/criterion-json-to-html.mjs (hunk 2 above)
Comment (jgil-r) [reply website/public/benchmarks/criterion-json-to-html.mjs:80]: Would you like me to change it back to how it was initially? If so, we will still get the following validation error: 
```
A numeric character reference expanded to carriage return
```

I was reading about the [title](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/title#multiline_titles) attribute and removing the extra spacing after the line break seems to give the right behavior.

Comment (strager) [reply website/public/benchmarks/criterion-json-to-html.mjs]: Your latest version looks good to me.

**Comment by strager** [comment]
It looks like there are merge conflicts. Rebase, then I'm happy to merge.

**Review by strager — APPROVED** [review]

File: website/public/install/index.ejs.html [hunk 4]
```diff
@@ -137,7 +137,12 @@
       }
 
       .long-shell-command-line {
+<<<<<<< HEAD
         /* 6 = "$ ".length + 4 spaces of indentation */
+=======
+        /* 4 = "$ ".length + 4 spaces of indentation */
+        display: inline-block;
+>>>>>>> fixed validation errors for /install page
```
Comment (strager) [inline website/public/install/index.ejs.html:145]: Blocking: Don't commit merge markers.

**Review by strager — CHANGES_REQUESTED** [review]

File: website/public/install/index.ejs.html (hunk 4 above)
Comment (jgil-r) [reply website/public/install/index.ejs.html]: Just removed them, thank you for you patience!

**Review by strager — APPROVED** [review]

**Pull request merged**